skins/oasis/css/core/ads.scss

```diff
@@ -203,7 +203,7 @@ $global-navigation-and-sitewide-messages-index: 5000101;
 
 /* grid layout transition temp code.  remove and integrate into main code after transition is complete */
 .WikiaGrid {
-	#TOP_RIGHT_BOXAD {
+	#TOP_BOXAD {
```

Comment:
Cache is going to hurt us 🙈

Reply:
Yeah, it will :/ Do you have an idea how to workaround this?

Reply:
Double styles, for a while

Reply:
Fixed.